Dockerfile

```diff
@@ -43,6 +43,9 @@ RUN curl -O "https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cl
 COPY wait-for-it.sh /usr/local/bin/
 COPY entrypoint.sh /usr/local/bin/
 
+RUN usermod -u 1000 www-data
+RUN groupmod -g 1000 www-data
+
```

Comment:
This PR is focused on debug logs, it would be nice if the Docker or any other kind of unrelated files were moved to a separate PR

Reply:
I can remove this.